framework/src/systems/NonlinearSystemBase.C

```diff
@@ -3355,6 +3355,13 @@ NonlinearSystemBase::setupScalingGrouping()
          ++group_index)
       for (const auto & var_name : _scaling_group_variables[group_index])
       {
+        if (find(_ignore_variables_for_autoscaling.begin(),
+                 _ignore_variables_for_autoscaling.end(),
+                 var_name) != _ignore_variables_for_autoscaling.end())
+          mooseError("Variable ",
+                     var_name,
+                     " cannot be in a scaling grouping and be also ignored via "
+                     "'ignore_variables_for_autoscaling'");
```

Comment:
I think you could do this check in `FEProblemSolve` and be able to give a `paramError` as opposed to a `mooseError`